datastreams/aggregator.go

```diff
@@ -115,10 +120,16 @@ func alignTs(ts, bucketSize int64) int64 { return ts - ts%bucketSize }
 
 func (a *aggregator) add(point statsPoint) {
 	btime := alignTs(point.timestamp, bucketDuration.Nanoseconds())
-	b, ok := a.buckets[btime]
+	var buckets map[int64]bucket
```

Comment:
instead of calling this function twice, we can call it once, but every time it's called, add the point to the two buckets (origin, and current)

Reply:
Actually the StatsPoint itself doesn't contain the pathway start times, so if we only create one statsPoints in the `pathway.go` code, at the aggregator we would need to recompute the "originTimestamp" by subtracting timestamp - pathwayLatency. That should be fine but might not be as accurate as sending out two statsPoints in the `pathway.go` code. Do you have a preference for one or the other?

Reply:
what do you think about adding originTimestamp to the `statsPoint`?

Reply:
Oh, actually, I believe you want to close this PR in favor of: https://github.com/DataDog/data-streams-go/pull/11/files?